### ISSUE TITLE
bug xcresulttool merge  command fails when scheme name has a space

### ISSUE THREAD

**Issue by jineshqa** [lyndsey-ferguson/fastlane-plugin-test_center#159]
### New Issue Checklist

- [X] Updated `fastlane-plugin-test_center` to the latest version
- [X] I read the [README.md](https://github.com/lyndsey-ferguson/fastlane-plugin-test_center/blob/master/README.md)
- [X] I reviewed the [example(s)](https://github.com/lyndsey-ferguson/fastlane-plugin-test_center/blob/master/README.md) for the action(s) I am using
- [X] I have removed any sensitive data such as passwords, authentication tokens, or anything else I do not want to world to see

### Issue Description

I am using multi_scan to run tests. Towards the end during result collation step fastlane fails if the scheme has space in the name.
```
Exit status of command 'xcrun xcresulttool merge /var/folders/_v/385bsl711hj8mg07gcv518y80000gn/T/XXX\ Xxxxxx\ QA-4.test_result20191022-27805-1xqu3dl.xcresult /var/folders/_v/385bsl711hj8mg07gcv518y80000gn/T/XXX\ Xxxxxx\ QA-4.test_result20191022-27805-1r4srkw.xcresult --output-path /var/folders/_v/385bsl711hj8mg07gcv518y80000gn/T/d20191022-27805-ii1mlg/XXX Xxxxxx QA.test_result' was 1 instead of 0.
error: unexpected argument Xxxxxx; use --help to list available arguments
```

I think if we could enclose the result of map in double quotes (") at below line it should resolve the issue.
https://github.com/lyndsey-ferguson/fastlane-plugin-test_center/blob/16599a56dad0f3e5f7b490c5123be20e9a27889e/lib/fastlane/plugin/test_center/actions/collate_test_result_bundles.rb#L60

This should be pretty straight forward. If you still need then verbose output please let me know. I'll modify my dummy project and get that.

**Comment by lyndsey-ferguson** [comment]
This [`tmp_collated_bundlepath`](https://github.com/lyndsey-ferguson/fastlane-plugin-test_center/blob/16599a56dad0f3e5f7b490c5123be20e9a27889e/lib/fastlane/plugin/test_center/actions/collate_test_result_bundles.rb#L62) needs to be `tmp_collated_bundlepath.shellescape`.

As my time is limited, could you please open up a PR to make that change?

**Comment by jineshqa** [comment]
Yes sir. On it.

**Comment by jineshqa** [comment]
Simulator log collection is failing for me when I run tests so I had to change scanfile from
`include_simulator_logs true`
to
`include_simulator_logs false`

**Comment by jineshqa** [comment]
@lyndsey-ferguson Please let me know if anything else needs to be done.

**Comment by lyndsey-ferguson** [comment]
@jineshqa regarding the simulator collection crashes: you have to update to the latest fastlane. I fixed it [here](https://github.com/fastlane/fastlane/pull/15440)

**Comment by lyndsey-ferguson** [comment]
Fixed in [v3.8.12](https://github.com/lyndsey-ferguson/fastlane-plugin-test_center/releases/tag/v3.8.12). Thanks @jineshqa!!

# 🎉
